src/platforms/java/guides/spring/advanced-usage.mdx

```diff
@@ -8,7 +8,9 @@ Record user information from an HTTP request or by registering a Spring bean for
 
 ## Recording User Information From HTTP Request
 
-To record the user's IP address and `Principal#name` as the username, set the personal information flag to `true`.
+To record the user's IP address and `Principal#name` as the username so you can then view in [Event Details](/product/performance/event-detail/:
```

Comment:
@maciejwalkowiak link is broken, you can see the preview via vercel https://sentry-docs-git-fork-maciejwalkowiak-gh-3482.sentry.dev/platforms/java/guides/spring/advanced-usage/